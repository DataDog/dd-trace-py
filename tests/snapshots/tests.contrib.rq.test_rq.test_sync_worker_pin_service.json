[[
  {
    "name": "rq.queue.enqueue_job",
    "service": "rq",
    "resource": "tests.contrib.rq.jobs.job_add1",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "worker",
    "error": 0,
    "meta": {
      "_dd.p.dm": "-0",
      "component": "rq",
      "job.func_name": "tests.contrib.rq.jobs.job_add1",
      "job.id": "d99c66b9-b664-4c42-a33d-541a7518cfa3",
      "language": "python",
      "queue.name": "q",
      "runtime-id": "2cbc4798a5e1448ea9627228c43b1739",
      "span.kind": "producer"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "process_id": 67067
    },
    "duration": 2164000,
    "start": 1667319043880113000
  },
     {
       "name": "rq.worker.perform_job",
       "service": "my-pin-svc",
       "resource": "tests.contrib.rq.jobs.job_add1",
       "trace_id": 0,
       "span_id": 2,
       "parent_id": 1,
       "type": "worker",
       "error": 0,
       "meta": {
         "_dd.p.dm": "-0",
         "component": "rq",
         "job.id": "d99c66b9-b664-4c42-a33d-541a7518cfa3",
         "job.origin": "q",
         "job.status": "finished",
         "language": "python",
         "runtime-id": "2cbc4798a5e1448ea9627228c43b1739",
         "span.kind": "consumer"
       },
       "metrics": {
         "_dd.top_level": 1,
         "_dd.tracer_kr": 1.0,
         "_sampling_priority_v1": 1,
         "process_id": 67067
       },
       "duration": 6290000,
       "start": 1667319043895995000
     },
        {
          "name": "rq.job.perform",
          "service": "my-pin-svc",
          "resource": "tests.contrib.rq.jobs.job_add1",
          "trace_id": 0,
          "span_id": 3,
          "parent_id": 2,
          "type": "",
          "error": 0,
          "meta": {
            "component": "rq",
            "job.id": "d99c66b9-b664-4c42-a33d-541a7518cfa3"
          },
          "duration": 900000,
          "start": 1667319043897216000
        }]]
