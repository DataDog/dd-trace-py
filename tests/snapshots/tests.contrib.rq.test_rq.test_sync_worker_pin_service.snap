[[{"name" "rq.queue.enqueue_job"
   "service" "rq"
   "resource" "tests.contrib.rq.jobs.job_add1"
   "type" "worker"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1630333148845191200
   "duration" 4423000
   "meta" {"runtime-id" "db58ccbfef044eeab2b2670b8b124068"
           "queue.name" "q"
           "job.id" "7f475cdd-3f19-4f23-86ad-c60a6d3dac0f"
           "job.func_name" "tests.contrib.rq.jobs.job_add1"}
   "metrics" {"_dd.agent_psr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 462
              "_dd.tracer_kr" 1.0}}
      {"name" "rq.worker.perform_job"
       "service" "my-pin-svc"
       "resource" "tests.contrib.rq.jobs.job_add1"
       "type" "worker"
       "error" 0
       "span_id" 1
       "trace_id" 0
       "parent_id" 0
       "start" 1630333148884054600
       "duration" 38824600
       "meta" {"runtime-id" "db58ccbfef044eeab2b2670b8b124068"
               "job.id" "7f475cdd-3f19-4f23-86ad-c60a6d3dac0f"
               "job.status" "finished"
               "job.origin" "q"}
       "metrics" {"_sampling_priority_v1" 1
                  "system.pid" 462
                  "_dd.tracer_kr" 1.0}}
          {"name" "rq.job.perform"
           "service" "my-pin-svc"
           "resource" "tests.contrib.rq.jobs.job_add1"
           "error" 0
           "span_id" 2
           "trace_id" 0
           "parent_id" 1
           "start" 1630333148888827500
           "duration" 2674200
           "meta" {"job.id" "7f475cdd-3f19-4f23-86ad-c60a6d3dac0f"}}]]
